docs/rfcs/RFC0002-MonoRepo-Gardener-Rotations.md

```diff
@@ -0,0 +1,137 @@
+> to ensure that PRs get reverted.
+
+### Prerequisites for gardening
+
+You will need to be part of the corresponding GitHub gardener team - this will be documented
+per repo.
+
+### Team member rotation
+
+Initially rotations will be hand selected. We are working on integrating with tools
+to make the current on-call more discoverable and alerts to be sent automatically to
+them.
+
+## Communication norms
+
+When responding to an incident, communicate early and often:
+
+1. Acknowledge ("ack") an issue where it is reported and announce the actions
+   you are taking or delegating clearly. Do this in the corresponding channel
+   you will be provided access to.
+2. When an issue or incident affects multiple developers or will take time to
+   resolve, consider announcing that to affected developers and filing a
+   tracking issue on GitHub.
+
+Primary communication about the changes in a commit should happen on the pull
+request that commit came from. This ensures that members of the open source
+community (who may not be members of Discord or Slack servers) can see the
+discussion and avoid duplicating effort.
+
+Secondary communication like real time discussion can occur on Discord, Slack,
+or Teams. Initially we will be leveraging Teams for these discussions.
+
+## Reacting to issues
+
+### Monitoring and alerting
+
+TBD - We will work to create alerting on corresponding channels.
+
+### Identify the source of the issue
+
+When CI is failing view the most recent job that passed. 
+Check PRs that were submitted between the two. If any PR bypassed presubmits
+it's an obvious candidate. Otherwise, this your candidate list. If no obvious
+change in this list, consult the gardener chat for more help if you can't
+figure it out yourself.
+
+It's also possible that it's not PR related and either broken infra or flaky test;
+see the following two sections.
+
+### Flaky tests
+
+If a test fails periodically, some options are:
+
+* De-flake the test
+* Disable the test or make it opt-in
+* Remove the test
+* Have the test run multiple times
+
+### Unhealthy CI machines
+
+We use both standard GitHub-hosted runners and a growing number of self-hosted
+runners across our projects. Sometimes the self-hosted runners need maintenance
+or troubleshooting to keep them running jobs successfully. See the section
+below for information about each type of self-hosted runner we use.
+
+### Reverting vs fixing-forward
+
+When a commit breaks a build, the first thought should be to revert that commit.
+
+* Reverts are generally safe, fast to make, and easy to review.
+* Fix-forwards can be tempting but carry additional risks and are necessarily
+  urgent to review.
+* When the shared `develop` branch is affected by a failing workflow, every
+  developer working off of that branch is affected. Reverting a change puts the
+  burden for fixing the issue back on just the original author of the change.
```

Comment:
May want to emphasize the timeline and SLO here or elsewhere. I'd like to set expectations for turnaround time on the order of say... 30 minutes - 2 hours? Maybe with an escalation path for longer durations.

Could use a `[!IMPORTANT]` alert or some other styling: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts